automation/addLabelsOnPullRequest.automation.js

```diff
@@ -65,103 +63,38 @@ exports.addMergedLabel = async context => {
 		const issueLabels = await listIssueLabels(context);
 
 		const foundApproveLabel = issueLabels.data.filter(
-			label => label.name === ':white_check_mark: Approved'
+			label => label.name === ":white_check_mark: Approved"
 		);
 		const foundWIPLabel = issueLabels.data.filter(
-			label => label.name === ':construction: WIP'
+			label => label.name === ":construction: WIP"
 		);
 
-		addLabel([':sparkles: Merged:'], context);
+		addLabel([":sparkles: Merged:"], context);
 
 		if (foundApproveLabel.length > 0) {
-			removeLabel([':white_check_mark: Approved'], context);
+			removeLabel([":white_check_mark: Approved"], context);
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹↹` with `······`](https://app.codacy.com/gh/aayushchugh/bot/pullRequest?prid=10060852)
```suggestion
      removeLabel([":white_check_mark: Approved"], context);
```